### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by kellerza** [kellerza/sunsynk#110] [Announcements]
A new documentation website is live - https://kellerza.github.io/sunsynk/

**Comment by helgard999** [comment]
Hi Johann
Thanks for all the effort and information that you make available.
Is it possible to create links to the mod-bus documents of the various inverters? I'm trying to understand how the master and slaves work together and handle accumulated energy. I have two 5kW Deye inverters each with a WiFi logger as they call it, so I can experiment a bit. Would like to to see "plant power, charge and energy" also instead of power and charge and energy from each individual inverters.

**Reply by kellerza** [reply]
#59 includes the Modbus register documents. These are not openly available form the manufacturers, but you can normally get them on a forum.

The master/slave will be 2 separate inverters. My guess is you will have to sum the data from the two. It might be better to discuss this on power-forum / energy-talk forums
